azure-application-insights-spring-boot-starter/build.gradle

```diff
@@ -44,8 +47,9 @@ sourceSets {
 }
 
 dependencies {
-    compile (project(':core'))
-    compile (project(':web'))
+    compile(project(':core'))
```

Comment:
So I checked the provided configuration and it says that client is required to add this dependency on the classpath which is not desired and hence I don't think it would work but I may be wrong and hence convinced.